api/mesh/v1alpha1/meshgateway/schema.yaml

```diff
@@ -0,0 +1,140 @@
+properties:
+  labels:
+    additionalProperties:
+      type: string
+    type: object
+  name:
+    type: string
+  spec:
```

Comment:
Ok one of the messy thing of the older policies was that the spec wasn't always nested inside a `spec` field.

For example `MeshGateway` on Universal is:

```
type: MeshGateway
name: the-gateway
mesh: default
labels:
  k8s.kuma.io/namespace: kuma-demo
  kuma.io/display-name: the-gateway
selectors:
  - match:
      kuma.io/service: gw-1
conf:
  listeners:
    - port: 8080
      protocol: HTTP
---
```

The same is the case with `Mesh`: https://github.com/kumahq/kuma/blob/b98468fbd6dcc43f597b99ec2f00cbda09b27599/test/framework/setup.go#L230-L251

Reply:
Universal was [decided to be out of scope for this](https://docs.google.com/document/d/1Ewe0yRx-vvAuQn5CWRFhJbBPLzdbBQ2wsQosZXk4Go8/edit?tab=t.0#heading=h.xx7061aqsfyv). Given that should I just create an issue or should we do something about it?

Reply:
But universal IS the api right? There's not rest API with universal entities including against Konnect

Reply:
You're 100% right - I don't know why but I thought they were all k8s because they had spec but of course they have spec in new policies 🤷 . Changed the code to not put things in `spec`.